src/components/DateSchedule.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import axios from "axios";
+import PropTypes from "prop-types";
+import { parseISO, format } from "date-fns";
+import styled from "styled-components";
+
+import { deleteAvailableDates } from "../features/counselorSlice";
+
+function DateSchedule({ onError }) {
+  const dispatch = useDispatch();
+  const userId = useSelector(({ user }) => user.data._id);
+  const scheduleList = useSelector(
```

Comment:
```suggestion
  const availableDates = useSelector(
```
현재 상태에서 가져오는 객체 이름으로 똑같이 변경하면 좋을 것 같네요!